### DISCUSSION TITLE
Do we need to import Vue APIs such as ref in script setup?

### DISCUSSION THREAD

**Discussion by ciarantr** [nuxt/nuxt#17857] [Questions] [answered]
Hi, I'm trying out the Nuxt 3 beta version and noticed when I set a variable as ref I do not need to import ref from vue  into my component.

for example
```
<script setup>
  const a = ref(true)
</script>

<template>
 {{ a }} // shows true
</template>
```

**Comment by wobsoriano** [comment, marked as answer]
They are auto-imported.

https://v3.nuxtjs.org/concepts/auto-imports

**Reply by ciarantr** [reply]
Thanks @wobsoriano  for the reply, must of missed that when I first read through the docs.
